doc/articles/uno-publishing-windows-unpackaged.md

```diff
@@ -0,0 +1,34 @@
+---
+uid: uno.publishing.windows.sideload.unpackaged.unsigned
+---
+
+# Publish an unpackaged app
+
+This guide will show how to create an unpackaged app using Windows App SDK.
+
+> [!IMPORTANT]
+> Building your app requires using the msbuild command (`dotnet build` is not compatible as of WinAppSDK 1.5).
+> [!NOTE]
+> Uno Platform also supports building apps for using the `net8.0-desktop` target framework, using Uno Platorm's own Skia Desktop implementation.
+
+## Build the app for publishing
+
+Packaging the app without a code signature allows for installing the app on a machine without having to install the signer's certificate. This portion of the guide is derived from the [official Windows App SDK documentation](https://learn.microsoft.com/en-us/windows/msix/package/unsigned-package).
+
+To publish your app:
+
+- Navigate to the folder of the app's `.csproj` (Building at the solution level is not supported)
+- Build your app using the following command:

```

Comment:
```suggestion
1. Navigate to the folder of the app's `.csproj` (Building at the solution level is not supported)
2. Build your app using the following command:

```
Maybe we could make this a numbered list? I know it has only two steps, but IMO is better for this case, I will let you decide